docs/DeveloperGuide.md

```diff
@@ -343,77 +343,208 @@ b. Require developers to download those libraries manually (this creates extra w
 
 Priorities: High (must have) - `* * *`, Medium (nice to have)  - `* *`,  Low (unlikely to have) - `*`
 
-
 Priority | As a ... | I want to ... | So that I can...
--------- | :-------- | :--------- | :-----------
-`* * *` | new user | see usage instructions | refer to instructions when I forget how to use the App
-`* * *` | user | add a new person |
-`* * *` | user | delete a person | remove entries that I no longer need
-`* * *` | user | find a person by name | locate details of persons without having to go through the entire list
-`* *` | user | hide [private contact details](#private-contact-detail) by default | minimize chance of someone else seeing them by accident
-`*` | user with many persons in the address book | sort persons by name | locate a person easily
+-------- | :----------- | :-------------------- | :-------------------------
+`* * *` | new user |  view an instruction manual | remember the format and syntax of commands when necessary
+`* * *` | user | quit the program from the command line | avoid the trouble of using the GUI when quit
+`* * *` | user with [events](#event)| add an event by specifying its start date/time and end date/time either in relative or absolute form i.e. `12 Mar` or `today`<br /><br /> **condition**: <br />when there is already an event booked for that time frame, I can be notified and choose whether or not to force the addition| book that time slot and be reminded of the event
+`* * *` | user with [tentative events](#tentative-event)| temporarily [block](#blocking) multiple time slots for a tentative event<br /><br /> **condition**: <br />I can later confirm its finalized time slot and [release](#releasing) all the other time slots| avoid time clash between events
+`* * *` | user with events| specify the location of an event | later adapt my actions based on where the event is located
+`* * *` | user with [tasks](#task)| add a task with a deadline, either in relative or absolute form | 
+`* * *` | user with tasks| add a task without a deadline | 
+`* * *` | user with tasks or events that are related to each other | add tags to tasks or events | easily search and categorize tasks or events
+`* * *` | user | add a description to a task or an event | refer to relevant information in the future
+`* * *` | user with tasks| specify the priority of a task | later compare the importance of tasks, which can be especially useful for comparing tasks without deadlines
+`* * *` | user with tasks| view only tasks with deadlines | 
```

Comment:
for these few user stories, the purpose is missing. a better way to write this would be:
`*as a* user ... *i should be* able to filter deadline ... *so that* i can view only deadlines ` or on those lines